packages/web-frontend/cypress/e2e/dashboardReports.spec.js

```diff
@@ -24,18 +24,16 @@ const closeDialogsBecauseOfCypressBug = () => {
 };
 
 describe('Dashboard reports', () => {
-  const testReportsForProject = (project, reports) => {
-    describe(`Project: ${project}`, () => {
-      const reportsByGroup = Cypress._.groupBy(reports, 'dashboardGroup');
-
+  const testReport = report => {
```

Comment:
Only a few lines is new code in this file: the rest are ordering functions to adhere to strict bottom-to-top definitions (functions that get called first are defined last etc)